src/pages/landing/weekend-trading/components/_hero.tsx

```diff
@@ -114,14 +114,35 @@ const TryButton = styled(LinkButton)`
     }
 `
 
-const HeroComponent = ({ title, content, background_data }) => {
+const query = graphql`
+    query {
+        p2p_hero_background: file(relativePath: { eq: "landing/weekend.png" }) {
+            ...fadeIn
+        }
+        p2p_hero_background_mobile: file(relativePath: { eq: "landing/weekend-m.png" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const Hero = () => {
+    const data = useStaticQuery(query)
+    const [is_mobile] = useBrowserResize()
+    const background = is_mobile ? data['p2p_hero_background_mobile'] : data['p2p_hero_background']
+
     return (
-        <BackgroundWrapper data={background_data}>
+        <BackgroundWrapper data={background}>
             <Wrapper p="4rem 0 0" justify="space-between" height="unset">
                 <InformationWrapper height="unset" direction="column">
-                    <StyledHeader type="hero">{title}</StyledHeader>
+                    <StyledHeader type="hero">
+                        {localize('Ride the trends even on weekends')}
+                    </StyledHeader>
                     <HeroContent direction="column" justify="flex-start">
-                        <Header type="subtitle-1">{content}</Header>
+                        <Header type="subtitle-1">
+                            {
+                                <Localize translate_text="Trade even when most financial markets are closed" />
+                            }
+                        </Header>
```

Comment:
Don't forget to add the semantics for the Headers.
```suggestion
                        <Header as="h2" type="subtitle-1">
                            {
                                <Localize translate_text="Trade even when most financial markets are closed" />
                            }
                        </Header>
```